name.abuchen.portfolio/src/name/abuchen/portfolio/model/Transaction.java

```diff
@@ -184,12 +185,19 @@ public Transaction(LocalDateTime date, String currencyCode, long amount, Securit
     {
         this();
         this.date = date;
+        this.exDate = date;
```

Comment:
sure that is `date `not `exDate`?

Reply:
I deleted the `exDate` field after @buchen's comment

>     * The `exDate` field has been added to the `Transaction` class. But in my understanding, this is only relevant for dividend payments - hence having the field on `AccountTransaction` should be enough.